### DISCUSSION TITLE
CTkTextbox

### DISCUSSION THREAD

**Discussion by Makenzuha** [TomSchimansky/CustomTkinter#1463] [Q&A] [answered]
Hello world, I have two questions
1) How to set the color of textbox;
2) How to change its size ( I don't want it to be long)
3) The font
I have read documentation but i can not understand these 3 points please help.
Thanks for reading.  
Source:
`        self.textbox = customtkinter.CTkTextbox(self, width=2,)
        self.textbox.insert("0.0", "Examples:\n\n" + "Examples1",)
        self.textbox.grid(row=0, column=1, padx=(15, 0), pady=(20, 0), sticky="nswe",)
        self.textbox.configure(state="disabled") 
        self.textbox.configure(font=customtkinter.CTkFont(family="<Roboto>",size=16, weight="bold"))

**Comment by bfedie518** [comment, marked as answer]
To configure a widget, you specify kwargs (keyword arguments) when initializing it. If you don't specify kwargs, you get the defaults.
```python
textbox = customtkinter.CTkTextbox(self)
```
vs
```python
textbox = customtkinter.CTkTextbox(self, width=50, fg_color='GG0000')
```

You can also configure it later by calling `configure()`.
```python
textbox.configure(font=('Tahoma', 12))
```
Your options are in the [arguments chart](https://github.com/TomSchimansky/CustomTkinter/wiki/CTkTextbox#arguments) in the documentation.

**Reply by Makenzuha** [reply]
thanks

**Comment by B16f00t** [comment]
I take advantage of the same thread, to ask since I would like to know how to put an image inside the TextBox.
I use this function:
self.chat_history.image_create(tk.END, image=self.user_images[user])
and it works in the tkinter textbox, but not in customtkinter,
I get this error:
embedding images is forbidden, because would be incompatible with scaling.

**Reply by bfedie518** [reply]
You can't. That error tells you everything you need to know.
